examples/TemplateModule/TemplateModule.psd1

```diff
@@ -0,0 +1,59 @@
+﻿#
+# Module manifest for module 'TemplateModule'
+#
+
+@{
+
+# Version number of this module.
+ModuleVersion = '1.0'
+
+# ID used to uniquely identify this module
+GUID = '42225364-811c-47de-8828-54f79d0aa599'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) 2016 Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Provides an example module which contains a Plaster template'
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @()
+
+# Variables to export from this module
+VariablesToExport = '*'
+
+# Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
+AliasesToExport = @()
+
+# List of all files packaged with this module
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
+PrivateData = @{
+
+    PSData = @{
+
+        Extensions = @(
+            @{
+                Module = "Plaster"

```

Comment:
Hmmm...  You've got a good point.  This would make the individual extension blocks shorter and more meaningful.  Though I suppose the downside is that there's the chance for naming conflicts, like if two people have their own format for "Snippets".  Once this logic is broken out into a separate module, we could keep a registry of extension types there, but that still wouldn't eliminate the problem entirely.

Thoughts?